src/CodeGen_LLVM.cpp

```diff
@@ -904,6 +904,8 @@ llvm::Type *CodeGen_LLVM::llvm_type_of(Type t) {
     return Internal::llvm_type_of(context, t);
 }
 
+llvm::TargetMachine *get_target_machine(const llvm::Module &module);
```

Comment:
+1 to that

On Thu, Apr 28, 2016 at 1:49 PM, Dillon Sharlet <notifications@github.com>
wrote:

> In src/CodeGen_LLVM.cpp
> <https://github.com/halide/Halide/pull/1170#discussion_r61498954>:
>
> > @@ -904,6 +904,8 @@ llvm::Type *CodeGen_LLVM::llvm_type_of(Type t) {
> >      return Internal::llvm_type_of(context, t);
> >  }
> >
> > +llvm::TargetMachine *get_target_machine(const llvm::Module &module);
>
> Maybe all of this stuff (get_md_*, clone_target_options,
> get_target_machine, etc.) all belongs in CodeGen_Internal.h/cpp? It's a
> bunch of basic LLVM helper stuff.
>
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
> <https://github.com/halide/Halide/pull/1170/files/8d9f16fd834fb6d8d490e8452b4edd4f07462517#r61498954>
>